exonum/src/node/consensus.rs

```diff
@@ -576,6 +559,35 @@ impl NodeHandler {
             self.remove_request(&RequestData::Transactions(hash));
             self.has_full_propose(hash, round);
         }
+        Ok(())
+    }
+
+    /// Handles raw transaction. Transaction is ignored if it is already known, otherwise it is
+    /// added to the transactions pool.
+    #[cfg_attr(feature = "flame_profile", flame)]
+    pub fn handle_tx(&mut self, msg: RawTransaction) {
+        //trace!("Handle transaction");
+        let tx = {
```

Comment:
Nitpick: braces seem to be superfluous in this case.

Reply:
Sure, I'll remove it.